libs/ui/tailwind.config.js

```diff
@@ -50,6 +50,22 @@ module.exports = {
         'separator-line-color': 'var(--separator-line-color)',
         'toggle-primary': 'var(--toggle-primary)',
         'toggle-active-switch': 'var(--toggle-active-switch)',
+
+        neutral: {
+          1: '#ffffff',
+          2: '#fcfcfc',
+          3: '#f5f5f5',
+          4: '#f0f0f0',
+          5: '#d9d9d9',
+          6: '#bfbfbf',
+          7: '#8c8c8c',
+          8: '#595959',
+          9: '#454545',
+          10: '#262626',
+          11: '#1f1f1f',
+          12: '#141414',
+          13: '#000000',
+        },
```

Comment:
allright, we might ping this when needed.
weird that you can use this tailwind colors outside package

Reply:
Tailwind config is shared both inside and outside the lib:

https://github.com/kodadot/nft-gallery/blob/47d18dd2136f2b86caa519c932c1a2211ff95d28/nuxt.config.ts#L17

Reply:
oh that's why, noted!